operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -286,7 +286,29 @@ export VM_TYPE="HotSpot"
 #  Copied from ${DOMAIN_HOME}/bin/startNodeManager.sh 
 export NODEMGR_HOME="${NODEMGR_HOME?}"
 export DOMAIN_HOME="${DOMAIN_HOME?}"
-export JAVA_OPTIONS="${JAVA_OPTIONS} -Dweblogic.RootDirectory=${DOMAIN_HOME}"
+
+# Apply JAVA_OPTIONS to Node Manager if NODEMGR_JAVA_OPTIONS not specified
+if [ -z ${NODEMGR_JAVA_OPTIONS} ]; then
+  NODEMGR_JAVA_OPTIONS="${JAVA_OPTIONS}"
+fi
+
+if [ -z "${NODEMGR_MEM_ARGS}" ]; then
+  # Default JVM memory arguments for Node Manager
+  NODEMGR_MEM_ARGS="-Xms64m -Xmx100m"
+fi
+
+# USER_MEM_ARGS is not applied to Node Manager.  Use NODEMGR_MEM_ARGS
+# (or NODEMGR_JAVA_OPTIONS) to specify JVM memory arguments for Node Manager.
+# Specifying a non-empty USER_MEM_ARGS environment variable prevents MEM_ARGS
+# from being specified in the common WebLogic environment scripts in the WebLogic
+# installation.  NOTE: Specifying USER_MEM_ARGS with ' ' (space, not empty string)
+# prevents WLS from calculating default values (see commBaseEnv.sh)
```

Comment:
Is the following a little more clear?  I'm trying to update the note to explain why
we're zeroing USER_MEM_ARGs...

We prevent USER_MEM_ARGS from being applied to the NM here and only pass
USER_MEM_ARGS to WL Servers via the WL props file above.  This is so that WL 
Servers and NM can have different tuning. Use NODEMGR_MEM_ARGS or
NODEMGR_JAVA_OPTIONS to specify JVM memory arguments for NMs.
 NOTE: Specifying USER_MEM_ARGS with ' ' (space, not empty string) 
prevents MEM_ARGS from being implicitly set by the WebLogic env
scripts in the WebLogic installation and WLS from inserting default 
values for memory arguments.  (See commBaseEnv.sh).